app/grandchallenge/participants/views.py

```diff
@@ -1,21 +1,26 @@
+from collections import defaultdict
+from typing import Any
```

Comment:
We do not run mypy on GC so no typing please.

Reply:
See https://mypy.readthedocs.io/en/stable/existing_code.html

> If your codebase is large, pick a subset of your codebase (say, 5,000 to 50,000 lines) and get mypy to run successfully only on this subset at first, _before adding annotations_.

Reply:
Check! I think this was a lazy copy-pasta, sorry!